plugins/utils/mark-radar.nix

```diff
@@ -0,0 +1,25 @@
+{ pkgs, config, lib, ... }:
+
+with lib;
+let
+  cfg = config.programs.nixvim.plugins.mark-radar;
+  helpers = import ../helpers.nix { inherit lib; };
+  defs = import ../plugin-defs.nix { inherit pkgs; };
+in
+{
+  options.programs.nixvim.plugins.mark-radar = {
+    enable = mkEnableOption "Enable mark-radar";
+  };
```

Comment:
I'd add the other mark-radar options here! Though I think it's fine to merge this for now, and then I can just add the rest.

Reply:
I'm not sure what other options could be added:
```
local opts = {
    set_default_mappings = true,
    highlight_group = "RadarMark",
    background_highlight = true,                    
    background_highlight_group = "RadarBackground", 
}
```
None of these seem like anything you'd want to override. And if you set `set_default_mappings` to false, there's no way to change it to something else.

Reply:
Sure, but I'd like to expose the options anyway just to be on the safe side of things, even if they are completely sensible defaults that 99% of people wouldn't want to mess with.

Reply:
Done.